hspec-core/src/Test/Hspec/Core/Config/Definition.hs

```diff
@@ -91,6 +91,7 @@ data Config = Config {
 , configFormatter :: Maybe V1.Formatter
 , configHtmlOutput :: Bool
 , configConcurrentJobs :: Maybe Int
+, configMaxTimePerTest :: Maybe Seconds
```

Comment:
Please move this closer to related options, e.g. after `configFailOnEmptyDescription` (matching the location in the help message).

Reply:
I try to keep names of command line options and the corresponding fields of the config data type consistent.  So this should be `configTimeout`, or if this is too generic then alternatively `configSpecItemTimeout`. I would probably go with the first.